files/packs.json

```diff
@@ -684,6 +684,24 @@
         "file": "SkyMobs.zip",
         "hash": "f226121d849d529fd2c23c47dd9b552b"
     },
+    {
+        "id": "nosplashtexts",
+        "display": "NoSplashTexts",
+        "creator": "BlockyRick, multiverison port by Erymanthus",
+        "description": "Multiversion resource pack to remove title screen splash text.",
+        "icon": "nosplashtexts.png",
+        "icon_scaling": "pixel",
+        "actions": [
+            {
+                "text": "Github",
+                "link": "https://github.com/RayDeeUx/NoSplashTexts/releases/download/v1.0.0/NoSplashTexts.zip"
```

Comment:
Why is the GitHub a direct download link instead of the project page

Reply:
gah!